superset/connectors/sqla/models.py

```diff
@@ -267,6 +269,28 @@ def get_sqla_col(self, label: Optional[str] = None) -> Column:
     def datasource(self) -> RelationshipProperty:
         return self.table
 
+    def get_extra_dict(self) -> Dict[str, Any]:
```

Comment:
this is copy pasted from the metric model right? Maybe we can have a `CertifiableMixin` that both models can use?

Reply:
I'll probably do this as a follow up just to get this going.